src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml

```diff
@@ -544,3 +544,12 @@ services:
             - '@prestashop.admin.sell.order.credit_slip.credit_slip_options_form_data_provider'
             - 'PrestaShopBundle\Form\Admin\Sell\Order\CreditSlip\CreditSlipOptionsType'
             - 'CreditSlip'
+
+    prestashop.admin.feature_flags.form_handler:
+        class: 'PrestaShop\PrestaShop\Core\Form\Handler'
+        arguments:
+            - '@form.factory'
+            - '@prestashop.core.hook.dispatcher'
+            - '@prestashop.admin.configure.advanced_parameters.feature_flags_form_data_provider'
+            - 'PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\FeatureFlag\FeatureFlagsType'
+            - 'ExperimentalFeature'
```

Comment:
Isn't `FeatureFlag` ?

Reply:
Oooooh nice catch the very last one! I missed it in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/b8459900f4df0c8803d43ff6bccecaa3439df55e

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/da43bcd2b47c0e946229794629a6d5d7ed1982da